enginetest/testdata.go

```diff
@@ -78,6 +78,30 @@ func createSubsetTestData(t *testing.T, harness Harness, includedTables []string
 	var table sql.Table
 	var err error
 
+	if includeTable(includedTables, "specialtable") {
+		wrapInTransaction(t, myDb, harness, func() {
+			table, err = harness.NewTable(myDb, "specialtable", sql.Schema{
+				{Name: "name", Type: sql.MustCreateStringWithDefaults(sqltypes.VarChar, 20), Source: "specialtable"},
+			})
+
+			if err == nil {
+				InsertRows(t, NewContext(harness), mustInsertableTable(t, table),
+					sql.NewRow("first_row"),
+					sql.NewRow("second_row"),
+					sql.NewRow("third_row"),
+					sql.NewRow(`%`),
```

Comment:
Need some test rows where there are characters other than the special characters